crates/requirements-txt/src/lib.rs

```diff
@@ -903,10 +945,14 @@ impl Display for RequirementsTxtFileError {
                     self.file.simplified_display(),
                 )
             }
-            RequirementsTxtParserError::Parser { message, location } => {
+            RequirementsTxtParserError::Parser {
+                message,
+                line,
+                column,
+            } => {
                 write!(
                     f,
-                    "{message} in `{}` at position {location}",
+                    "{message} in `{}` at position {line}:{column}",
```

Comment:
Could you make that format `at <REQUIREMENTS_TXT>:<line>:<col>`? This format is support by many IDEs

Reply:
Changed in 223fd99ac1b44e0557227d1c461a20bf1a9d85f2

Reply:
Note, mileage may vary, I noticed that for full-paths it does highlight on my IDE, but not relative paths like `./requirements.txt`. We could try to canonicalize always?

Reply:
I think using `current_dir()` and joining the path on top of it is the best way, canonicalize could be a path outside the apparent project root (we used to canonicalize everything but rolled that back since it caused problems where software expected the "virtual" name of the link).